charts/open-notificaties/templates/_helpers.tpl

```diff
@@ -99,3 +99,35 @@ Worker selector labels
 app.kubernetes.io/name: {{ include "open-notificaties.workerName" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Create a name for Flower
+We truncate at 56 chars in order to provide space for the "-flower" suffix
```

Comment:
```suggestion
We truncate at 57 chars in order to provide space for the "-flower" suffix
```

Reply:
Will fix this in a separate commit, as I see it is also faulty for the `-worker` part.

Reply:
Fixed in bd7932dd759bcb4328f2f9c4e20e367004b5d697